### PR TITLE
Add changelog for release 2019-04-12-1

### PR DESCRIPTION
**Pull request by govindkrjoshi**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,46 @@
...
+- Fix misplaced assignment while setting patients status to dead
+- Fixed facility link paths
+- Fix recorded patients count on the dashboard
+### Deprecated
+### Removed
+### Security
+
+## Api
+### Added
+- ExotelAPI to fetch call_details
```
Comment (divs1210) [inline CHANGELOG.md]: this is not part of our simple API, though

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,7 +6,46 @@
...
+- Populate the CallLog result from the terminate response
```
Comment (divs1210) [inline CHANGELOG.md]: We don't need to add this

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,7 +6,46 @@
...
+- Adding the sidekiq monitoring route for owners only
+- Add connection pooling and redis store for CallSession
+- Add support to run sidekiq as a systemctl service
```
Comment (divs1210) [inline CHANGELOG.md]: these things don't look API related

File: CHANGELOG.md (hunk 1 above)
Comment (govindkrjoshi) [reply CHANGELOG.md]: These are changes related to API behavior no?

File: CHANGELOG.md (hunk 3 above)
Comment (govindkrjoshi) [reply CHANGELOG.md]: This is related since sidekiq will be used for both API and Portal. Chose to put the changelog in API.

File: CHANGELOG.md (hunk 1 above)
Comment (divs1210) [reply CHANGELOG.md]: This is us calling the Exotel API to fetch call related data. Not Simple API change. Can rephrase it to say Exotel API wrapper to fetch call details from Exotel.
